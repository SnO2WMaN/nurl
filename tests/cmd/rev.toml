args = [
    "https://github.com/nix-community/nurl",
    "7d7ed2f239f2c77e26a724d7c786e8079407c767",
]
