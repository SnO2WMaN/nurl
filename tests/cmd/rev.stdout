fetchFromGitHub {
  owner = "nix-community";
  repo = "nurl";
  rev = "7d7ed2f239f2c77e26a724d7c786e8079407c767";
  hash = "sha256-TtH0sfWFWe3oYK/8jJslqjrEY5rR7HGAVDD5iQ2+spY=";
}